frappe/integrations/utils.py

```diff
@@ -35,21 +35,44 @@ def make_post_request(url, **kwargs):
 def make_put_request(url, **kwargs):
 	return make_request('PUT', url, **kwargs)
 
-def create_request_log(data, integration_type, service_name, name=None, error=None):
-	if isinstance(data, str):
-		data = json.loads(data)
-
-	if isinstance(error, str):
-		error = json.loads(error)
+def create_request_log(
+	data,
+	integration_type=None,
+	service_name=None,
+	name=None,
+	error=None,
+	request_headers=None,
+	output=None,
+	**kwargs
+):
```

Comment:
If this doesn't seem like worth maintaining, we can just keep the existing function as-is for other apps and create a new util called `create_integration_request` for future use.